src/components/common/Dialogs/CreatePaymentDialog/CreatePaymentDialog.tsx

```diff
@@ -94,9 +116,9 @@ const CreatePaymentDialog = ({
       <Form<FormValues>
         defaultValues={{
           forceAction: false,
-          fromDomain: filteredDomainId || Id.RootDomain,
+          fromDomainId: filteredDomainId || Id.RootDomain,
           recipient: undefined,
-          amount: '',
+          amount: 0,
```

Comment:
Just curious of the reasoning behind changing this from empty string to 0? Also I noticed the validation message "amount is a required field" is no longer present if the amount field is empty.

Reply:
Sure, we are validating the `amount` field as a number in the validation schema, and we infer the `FormValues` type from the schema, so the default value needs to be `0`, otherwise we get a typescript error.

Right now if you leave the field empty you should get an `Amount must be greater than 0`. I think that may be because `toFinite` interprets `''` as `0`.